gui/src/app/components/machine-view/add-machine/add-machine.js

```diff
@@ -48,7 +53,9 @@ class MachineViewAddMachine extends React.Component {
       return;
     }
     const selectedMachine = state.selectedMachine;
+    let constraints;
```

Comment:
lets set default values for `constraints` and `series`

Reply:
To `null`?